src/commandHelper.ts

```diff
@@ -0,0 +1,21 @@
+import chalk = require('chalk');
+
+export interface LogOptions {
+  indent?: number;
+  color?: string;
+  bold?: boolean;
+}
+
+export const format = (message: string, options?: LogOptions) => {
+  let formattedMessage = message;
+  if (options?.indent && options.indent > 0) {
```

Comment:
nit: just wanted to make sure you were aware of [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)

```javascript
if (options?.indent > 0) {
```